doc/sphinx/compiling.rst

```diff
@@ -39,6 +40,23 @@ Linux
 
       cython python-dev python-numpy python-numpy-dev python-setuptools
 
+* Building the python3 module requirs the following libraries::
+    
+      python3 python3-setuptools cython3 python3-numpy
+
+* For Fedora (version 22, or higher) users, the following packages should
+  be installed via the package manager::
+
+      g++ python scons sundials
```

Comment:
I think you need `sundials-devel` in order to get the header files for Sundials as well, right?